js/views/reflowable_view.js

```diff
@@ -693,7 +693,7 @@ var ReflowableView = function(options, reader){
         _$htmlBody.css('padding', 0);
 
         _paginationInfo.rightToLeft = _spine.isRightToLeft();
-
+        _paginationInfo.pageOffsetSize = _htmlBodyIsVerticalWritingMode ? _lastViewPortSize.height : (_lastViewPortSize.width + _paginationInfo.columnGap);
```

Comment:
Interesting, `_htmlBodyIsVerticalWritingMode ? _lastViewPortSize.height : (_lastViewPortSize.width + _paginationInfo.columnGap)` replaces `(_paginationInfo.columnWidth + _paginationInfo.columnGap) * _paginationInfo.visibleColumnCount` (see `onPaginationChanged_()` `_paginationInfo.pageOffset = _paginationInfo.pageOffsetSize * _paginationInfo.currentSpreadIndex;` in the source above) where `_paginationInfo.columnWidth` was `Math.round(((_htmlBodyIsVerticalWritingMode ? _lastViewPortSize.height : _lastViewPortSize.width) - _paginationInfo.columnGap * (_paginationInfo.visibleColumnCount - 1)) / _paginationInfo.visibleColumnCount);` ... I have to look at the arithmetics to make sure this does not affect vertical writing mode, or some other column-gap related edge-case computations :)